src/main.zig

```diff
@@ -1467,6 +1438,11 @@ fn buildOutputType(
                                 mem.eql(u8, linker_arg, "-static"))
                             {
                                 force_static_libs = true;
+                            } else if (mem.eql(u8, linker_arg, "--subsystem")) {
+                                const next_arg = split_it.next() orelse {
```

Comment:
```suggestion
                                const next_arg = args_iter.next() orelse {
```

so that we are consistent with other linker args such as `--entry` or `--stack`. Lemme know what you think!

Reply:
Just realised you asked about this. Yeah, this will swap out `,` for `=` making it more consistent across the additional args IMHO.

Reply:
Please ignore my earlier code change suggestion - I misread where it was in `main.zig`. However, I'd still prefer we had `--subsystem=<...>` - as far as I can remember, you just need to defer parsing this arg until you collect them all into `linker_args` container.

Reply:
Sounds good, `—-subsystem=x` it is, stay tuned!

Reply:
@kubkon memory serves you well, fixed in b2344cc18e9fa5dfa1d19025caa2b0f583498734!